features/content_publish.feature

```diff
@@ -3157,3 +3151,390 @@ Feature: Content Publishing
         ]
       }
       """
+
+    @auth
+    Scenario: Send correction with new featuremedia which does not fill all renditions
+      Given the "validators"
+        """
+        [
+            {"_id": "publish_text", "act": "publish", "type": "text", "schema":{}},
+            {"_id": "correct_text", "act": "correct", "type": "text", "schema":{}},
+            {"_id": "publish_picture", "act": "publish", "type": "picture", "schema":{}},
+            {"_id": "correct_picture", "act": "publish", "type": "picture", "schema":{}}
```

Comment:
This action should be 'correct', i.e.
`{"_id": "correct_picture", "act": "correct", "type": "picture", "schema":{}}`